integration_tests/src/main/python/string_test.py

```diff
@@ -994,6 +994,26 @@ def test_regexp_replace_word():
         ),
         conf=_regexp_conf)
 
+def test_predefined_character_classes():
+    gen = mk_str_gen('[a-zA-Z]{0,2}[\r\n!"#$%&\'()*+,-./:;<=>?@[\\]^_`{|}~]{0,2}[0-9]{0,2}')
```

Comment:
Could you update the fuzz tests in `RegularExpressionTranspilerSuite` to make sure all of the characters here are included in the generated data